ehitchhiking-ui/src/app/pages/main-screen/main-screen-component/main-screen.component.sass

```diff
@@ -23,9 +23,9 @@
   background-color: $red
 .save-route-btn
   position: fixed
-  right: 2.5rem
-  top: 4rem
+  right: 6.5rem
+  bottom: 5rem
```

Comment:
position: fixed
right: 6.5rem
bottom: 5rem
are common styles for save-route-btn and view-routes-btn, pls create shared class for them

Reply:
fix